expression/visitor.go

```diff
@@ -107,8 +107,8 @@ type Visitor interface {
 	// VisitFunctionTrim visits FunctionTrim expression.
 	VisitFunctionTrim(v *FunctionTrim) (Expression, error)
 
-	// VisitDateAdd visits DateAdd expression.
-	VisitDateAdd(da *DateAdd) (Expression, error)
+	// VisitDateArith visits DateAdd and DateSub expression.
```

Comment:
This comment can update to `VisitDateArith visits DateArith expression.`, for we only have DateArith expression now.